.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If you want *faster* PR reviews, read how: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md
+-->
+
+**What this PR does / why we need it**:
+
+**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
+
+**Special notes for your reviewer**:
+
+**If applicable**:
+- [ ] documentation
+- [ ] unit tests
+- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)
+
+**Release note**:
```

Comment:
I don't think we really leverage the release note, so we may not want to put it here. Let's see what some of the other maintainers here

Reply:
Ok